packages/blockchain-wallet-v4-frontend/src/data/components/identityVerification/selectors.ts

```diff
@@ -37,6 +37,9 @@ export const getUserAddresses = (state: RootState) =>
 export const getUserRetrieveAddress = (state: RootState) =>
   state.components.identityVerification.userRetrieveAddress
 
+export const getIsNameValid = (state: RootState) =>
```

Comment:
Do we need this selector and `isNameValid` state at all? It does not seem to be used.